### ISSUE TITLE
Bug with Targeted SNAP review selection

### ISSUE THREAD

**Issue by courtrightd** [MN-Script-Team/DHS-MAXIS-Scripts#2884]
Script is having an error when trying to access PRIV cases, which potentially results in stopping midway through a multi-hour county-wide run.

**Comment by C-Love** [comment]
@courtrightd  - do you have a fix for this?

**Comment by courtrightd** [comment]
Yes I have it.  Is pull into the collaborative branch the process?

**Comment by C-Love** [comment]
I think since we flagged this as critical - you can pull it in to Master.
